script.js

```diff
@@ -0,0 +1,35 @@
+var numTurns = 0;
+
+$(document).ready(function(){
+
+  function buildBoard(){
+    for (i=0; i < 9; i++){
+      var addTile= $("<div class='tile'></div>");
+      $("#gameboard").append(addTile);
+    }
+  }
+
+  var clickies = function(){
```

Comment:
Why did you switch between `function name()` and `var name = function()`? I'd say stick with one or the other.